mono/metadata/boehm-gc.c

```diff
@@ -1649,69 +1682,66 @@ handle_data_first_unset (HandleData *handles)
 	return -1;
 }
 
-/* Returns the index of the current slot in the bitmap. */
-static void
-handle_data_grow (HandleData *handles, gboolean track)
+static MonoGCHandle
+mono_gchandle_tag (MonoGCHandle handle)
```

Comment:
Nit - `mono_gchange_tag_strong`?  So it's clear why it's being tagged.

Reply:
Done